dom-event-delegation/main.js

```diff
@@ -0,0 +1,10 @@
+var $taskList = document.querySelector('.task-list');
+function test(e) {
```

Comment:
You should not be naming anything `test`. It's not descriptive of what the function actually does

Reply:
my bad i forgot to change it after i was done testing out the first step of the exercise